UPGRADE-4.2.md

```diff
@@ -1,6 +1,27 @@
 UPGRADE FROM 4.1 to 4.2
 =======================
 
+Form
+----
+
+ * Deprecated calling `FormRenderer::searchAndRenderBlock` for fields which were already rendered. 
+   Instead of expecting such calls to return empty string, check if the field has already been rendered.
+ 
+   Before:
+   ```twig
+   {% for field in fieldsWithPotentialDuplicates %}
+      {{ form_widget(field) }}
+   {% endfor %}
+   ```
+   
+   After:
+   ```twig
+   {% for field in fieldsWithPotentialDuplicates %}
+     {% if not field.rendered %}
+      {{ form_widget(field) }}
+     {% endif %}
+   {% endfor %}
+   ```
```

Comment:
This?

> You are calling "form_%s" for field "%s" which has already been rendered before, trying to render fields which were already rendered is deprecated since Symfony 4.2 and will throw an exception in 5.0.

I think there is too much information in there for one sentence.